go-dlv.el

```diff
@@ -43,7 +43,7 @@
 ;; Sample marker line:
 ;; > main.main() ./test.go:10 (hits goroutine(5):1 total:1)
 (defvar go-dlv-marker-regexp
-  "^> .+(.*) \\(.+\\)\\:\\([0-9]+\\) (hits.+)$")
+  "^> .+(.*) \\(.+\\)\\:\\([0-9]+\\).*?$")
```

Comment:
I think `.*?` could just be `.*`, and more generally, `.*?$` could be removed altogether. But it works, so I am merging ;)